examples/v2/project_creation/config.yaml

```diff
@@ -32,9 +32,7 @@ resources:
       bindings:
       - role: roles/owner
         members:
-        # Make sure to include the DM service account that is doing the deployment!
```

Comment:
Include a comment saying that the service account making the project will automatically be added as an owner.